lib/datasource/github-releases/index.spec.ts

```diff
@@ -52,5 +56,19 @@ describe('datasource/github-releases', () => {
       ).toBe(false);
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('supports ghe', async () => {
+      const lookupName = 'some/dep';
+      httpMock
+        .scope(githubEnterpriseApiHost)
+        .get(`/api/v3/repos/${lookupName}/releases?per_page=100`)
+        .reply(200, responseBody);
+      const res = await github.getReleases({
+        registryUrl: 'https://git.enterprise.com',
+        lookupName,
+      });
```

Comment:
We can use `getPkgReleases` instead, but then we have to define `rangeStrategy`. If we need to define `rangeStrategy`, then we need to be consistent and refactor `github-tags` too.

Reply:
Are you sure? 

https://github.com/renovatebot/renovate/blob/e4fafdc3ac1b2dec18f417464cfc19de9b2de16a/lib/datasource/index.ts#L234

https://github.com/renovatebot/renovate/blob/e4fafdc3ac1b2dec18f417464cfc19de9b2de16a/lib/datasource/common.ts#L23-L29

Maybe you're thinking `versioning` ?

Reply:
Anyway, it's ok

Reply:
I think it's ok here to use direct call. Otherwise we need to pass `datasource` and `depName`. So This is easier testing here.